WSL/basic-commands.md

```diff
@@ -1,29 +1,33 @@
 ---
 title: Basic commands for WSL
 description: Reference for the basic commands included with Windows Subsystem for Linux (WSL).
-ms.date: 03/04/2022
+ms.date: 11/16/2022
 ms.topic: article
 ---
 
 # Basic commands for WSL
 
-The WSL commands below are listed in a format supported by PowerShell or Windows Command Prompt. To run these commands from a Bash / Linux distribution command line, you must replace `wsl` with `wsl.exe`.
+The WSL commands below are listed in a format supported by PowerShell or Windows Command Prompt. To run these commands from a Bash / Linux distribution command line, you must replace `wsl` with `wsl.exe`. For a full list of commands, run `wsl --help`. 
 
 ## Install
 
 ```powershell
 wsl --install
 ```
 
-Install WSL and the Ubuntu distribution of Linux. [Learn more](./install.md).
+Install WSL and the default Ubuntu distribution of Linux. [Learn more](./install.md). You can also use this command to install additional Linux distributions by running `wsl --install <Distribution Name>`. For a valid list of distribution names, run `wsl --list --online`. 
 
-## Install a specific Linux distribution
+Options include: 
 
-```powershell
-wsl --install --distribution <Distribution Name>
-```
+- `--distribution`: Specify the Linux distribution to install. You can find available distributions by running `wsl --list --online`. 
+- `--no-launch`: Install the Linux distribution but do not launch it automatically.
+- `--web-download`: Install from GitHub rather than using the Microsoft Store. *(This will result in not receiving as-available updates from the store.)*
```

Comment:
Is that actually the case ? Not sure if the store might not be able of recognizing the package even if installed via GH